component-tools/src/test/java/org/talend/sdk/component/tools/StudioInstallerTest.java

```diff
@@ -144,7 +144,7 @@ private void assertSetup(File studioHome) throws IOException {
         final File configIni = new File(studioHome, "configuration/config.ini");
         assertEquals(
                 singleton("component.java.registry="
-                        + registration.getAbsolutePath().replace("\\", "\\\\").replace(":", "\\:")),
+                        + registration.getAbsolutePath().replace("\\", "/").replace("\\", "\\\\").replace(":", "\\:")),
```

Comment:
Your tests on windows do not pass? AFAIK forward slashes work on windows.

Reply:
In StudioInstaller class(https://github.com/Talend/component-runtime/blob/master/component-tools/src/main/java/org/talend/sdk/component/tools/StudioInstaller.java#L162) we generate the registry location and replace all "\" symbols with "/". Then we put it to the config.ini class(https://github.com/Talend/component-runtime/blob/master/component-tools/src/main/java/org/talend/sdk/component/tools/StudioInstaller.java#L174-L176). Thus we need to do the same in the test to check the correct location. It doesn't really matter which symbol is used in windows, but if we change StudioInstaller it might break system in Mac or Linux, thus I chose to change the test instead of changing the code of StudioInstaller class.